src/ec/packages/ec-component-file/file.html.twig

```diff
@@ -0,0 +1,112 @@
+{% spaceless %}
+{#
+  Parameters:
+  - "icon" (object) (default: {}): object of type Icon; file type
+  - "title" (string) (default: '')
+  - "language" (string) (default: '')
+  - "meta" (string) (default: '')
+  - "download" (object) (default: {}): object of type Link
+  - "translation" (array) (default: []):
+    - "toggle" (object) (default: {}): object of type Button
+    - "items" (array) (default: []):
+      - "title" (string) (default: '')
+      - "meta" (string) (default: '')
+      - "lang" (string) (default: '')
+    - "description (string) (default:'')
+  - "extra_classes" (optional) (string) (default: '') Extra classes (space separated) for the icon
+  - "extra_attributes" (optional) (array) (default: []) Extra attributes for icon
+    - "name" (string) Attribute name, eg. 'data-test'
+    - "value" (string) Attribute value, eg: 'data-test-1'
+#}
+
+{# Internal properties #}
+
+{% set _icon = icon|default({}) %}
+{% set _title = title|default('') %}
+{% set _language = language|default('') %}
+{% set _meta = meta|default('') %}
+{% set _download = download|default({}) %}
+{% set _translation = translation|default([]) %}
+
+{% set _css_class = 'ecl-file' %}
+{% set _extra_attributes = '' %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }} data-ecl-file>
+  <div class="ecl-file__container">
+    {%- include '../ec-component-icon/icon.html.twig' with _icon|merge({extra_classes: 'ecl-file__icon'}) -%}
+
+    <div class="ecl-file__info">
+      <div class="ecl-file__title">{{ title }}</div>
+      <div class="ecl-file__language">{{ language }}</div>
+      <div class="ecl-file__meta">{{ meta }}</div>
+    </div>
+
+    {%- include '../ec-component-link/link.html.twig' with _download|merge({
+      link: _download.link|default({})|merge({
+        type: 'standalone',
+        icon_position: 'after',
+      }),
+      extra_classes: 'ecl-file__download ecl-button ecl-button--secondary',
```

Comment:
`ecl-button` comes by default, and `--secondary` would be better passed through `variant`.
Actual extra class is `ecl-file__download`

Reply:
well unfortunately, that's not true.
That's a link styled as a button, so these classes are needed.
As far as I know this is the only place where we use this kind of "metamorph" component